docs/en_US/Compression/v2_pruning_algo.rst

```diff
@@ -397,3 +398,43 @@ User configuration for Simulated Annealing Pruner
 **PyTorch**
 
 .. autoclass:: nni.algorithms.compression.v2.pytorch.pruning.SimulatedAnnealingPruner
+
+Auto Compress Pruner
+--------------------
+
+For each round, AutoCompressPruner prune the model for the same sparsity to achive the overall sparsity:
```

Comment:
"for the same ..." -> for a fixed sparsity ratio (e.g., 20%) to achieve the overall sparsity (e.g., 80%).

Reply:
change to

For total iteration number :math:`N`, AutoCompressPruner prune the model that survive the previous iteration for a fixed sparsity ratio (e.g., :math:`1-{(1-0.8)}^{(1/N)}`) to achieve the overall sparsity (e.g., :math:`0.8`)